crates/oq3_semantics/src/syntax_to_semantics.rs

```diff
@@ -855,6 +839,14 @@ fn from_expression_list(
     asg::ExpressionList::new(inner_expression_list(expression_list, context))
 }
 
+fn from_qubit_list(qubit_list: Option<synast::QubitList>, context: &mut Context) -> Vec<asg::TExpr> {
+    // Because None means a coding error upstream. Better to blow up here.
```

Comment:
```suggestion
     // Warning, I think map overlooks None. This can cause a bug in the present case.
    // Because None means a coding error upstream. Better to blow up here.
```